src/config/supported_distros.py

```diff
@@ -1,24 +1,24 @@
 SUPPORTED_DISTROS = {
 'IBM z/OS ™': {
-	'z/OS Software': 'ZOS_Software_List.json'
+	'z/OS Software': 'ZOS_Software_List'
 },
 'IBM Z and LinuxONE Container Registry': {
-	'Container Images for IBM Z and LinuxONE': 'IBMZ-container-registry.json'
+	'Container Images for IBM Z and LinuxONE': 'IBMZ_container_registry'
```

Comment:
Can you create a separate PR to change this to IBMZ_container_registry.json first?

This data source is created via a script run by a team at IBM, so I need to reach out to them to change the name, so their next PR doesn't just change it back :smile: And that allows us to approve the other PR you made in the data directory more quickly than the rest.